### ISSUE TITLE
`babel/arrow-parens` is broken

### ISSUE THREAD

**Issue by mdziekon** [babel/eslint-plugin-babel#15]
This code breaks `babel/arrow-parens` rule:
```javascript
const test = async (something) => {
    return something * 42;
};

test();
```

with this error
```javascript
1:14  error  Expected parentheses around arrow function argument  babel/arrow-parens
```

My .eslintrc file: http://pastebin.com/zJW5KYTa

It should not throw this error.

**Comment by mdziekon** [comment]
False alarm, I just had to update to the most recent ``babel-eslint`` version (which is 4.1.1 for now).
Sorry for any trouble.
